crates/factor-outbound-mysql/src/host.rs

```diff
@@ -38,8 +40,15 @@ impl<C: Client> v2::Host for InstanceState<C> {}
 
 #[async_trait]
 impl<C: Client> v2::HostConnection for InstanceState<C> {
-    #[instrument(name = "spin_outbound_mysql.open_connection", skip(self), err(level = Level::INFO), fields(otel.kind = "client", db.system = "mysql"))]
+    #[instrument(name = "spin_outbound_mysql.open", skip(self, address), err(level = Level::INFO), fields(otel.kind = "client", db.system = "mysql", db.address = Empty, server.port = Empty, db.namespace = Empty))]
     async fn open(&mut self, address: String) -> Result<Resource<Connection>, v2::Error> {
+        if let Ok(url) = Url::parse(&address) {
+            let span = tracing::Span::current();
+            span.record("db.address", url.host_str().unwrap_or_default());
```

Comment:
Should this be `db.host` instead of `db.address` or is there some other external reason to label this as a address?

Reply:
Following the guidelines [here](https://opentelemetry.io/docs/specs/semconv/database/sql/).